src/main/js/account/account-service.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Account service to abstract the logic to interact with the bank account API.
+ *
+ * @author Xiaobing Hou
+ * @date 06/03/2022
+ */
+
+import axios from "axios";
+
+const PATH = "/api/v1/bank-accounts";
+
+class AccountService {
+    /**
+     * To get all bank accounts .
+     *
+     * @author Xiaobing Hou
+     * @date 06/03/2022
+     */
+    findAllBankAccount = () => {
+        const token = sessionStorage.getItem('trackrToken')
+        console.log(token)
+
+        axios.interceptors.request.use(function (config) {
+            config.headers.Authorization = "Bearer " + token;
+            return config;
+        })
```

Comment:
Need to remove as the token is being submitted an HTTP only Cookie. The motivation is that localStorage is not secure and another site can load some JS code into your browser and steal the token from localStorage then submit request on your behalf.

Reply:
Done